### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by nochinator** [redeclipse/base#1566] [General]
I just noticed that Red Eclipse does not have a license for it git repository or in its distribution and no mention is made of what licence the source code is under, only that it is "open source". If it is not actually under any license then it would be a good idea to select one, if it is then might I suggest adding a LICENSE file in the root directory of the repository.

**Comment by IceflowRE** [comment]
https://github.com/redeclipse/base/blob/master/doc/license.txt

**Reply by IceflowRE** [reply]
But it is indeed a bit hidden.

**Reply by qreeves** [reply]
Readme used to state the licence

**Reply by nochinator** [reply]
It might be a good idea to add the license in a "LICENSE" file at the root of the project.
